test/fake-timers-test.js

```diff
@@ -3775,11 +3778,26 @@ describe("FakeTimers", function () {
                 delete Performance.prototype.someFunc3;
             });
 
+            it("should mock performance on Node 16+", function () {
+                // node 16+ has a performance object but not a global constructor
+                if (typeof performance === "undefined") {
+                    this.skip();
```

Comment:
shouldn't there be a `return` after the `skip`s so we don't do further works that's ignored?

Reply:
I'm... not sure should there?

Reply:
No other places in the code do this - so I guessing "no" but I'm just cargo-culting here

Reply:
Either `return` or use a `if/else`. Ref the [Mocha docs:](https://mochajs.org/#:~:text=Best%20practice%3A%20To%20avoid%20confusion%2C%20do%20not%20execute%20further%20instructions%20in%20a%20test%20or%20hook%20after%20calling%20this.skip().)
> _Best practice:_ To avoid confusion, do not execute further instructions in a test or hook after calling this.skip().

I can fix this.